support/globalconfig/infrastructure.go

```diff
@@ -57,5 +57,8 @@ func ReconcileInfrastructure(infra *configv1.Infrastructure, hcp *hyperv1.Hosted
 	case hyperv1.IBMCloudPlatform:
 		infra.Status.PlatformStatus = &configv1.PlatformStatus{}
 		infra.Status.PlatformStatus.Type = configv1.IBMCloudPlatformType
+	case hyperv1.NonePlatform:
+		infra.Status.PlatformStatus = &configv1.PlatformStatus{}
```

Comment:
Please just set
`infra.Status.PlatformStatus.Type = configv1.PlatformType(hcp.Spec.Platform.Type)`
for any every case, no need to discriminate by platform.

Reply:
Ack thanks, I see that's already the default so I'll remove setting the Type

Reply:
we can drop ibm/none specifics and just do:

```
infra.Status.PlatformStatus = &configv1.PlatformStatus{}
infra.Status.Platform = configv1.PlatformType(hcp.Spec.Platform.Type)

switch hcp.Spec.Platform.Type {
	case hyperv1.AWSPlatform:
		infra.Spec.PlatformSpec.AWS = &configv1.AWSPlatformSpec{}
		infra.Status.PlatformStatus.Type = configv1.AWSPlatformType
		infra.Status.PlatformStatus.AWS = &configv1.AWSPlatformStatus{
			Region: hcp.Spec.Platform.AWS.Region,
		}
		tags := []configv1.AWSResourceTag{}
		for _, tag := range hcp.Spec.Platform.AWS.ResourceTags {
			tags = append(tags, configv1.AWSResourceTag{
				Key:   tag.Key,
				Value: tag.Value,
			})
		}
		infra.Status.PlatformStatus.AWS.ResourceTags = tags
```

I'll do that in a follow up PR.